components/var_dumper.rst

```diff
@@ -105,6 +105,16 @@ This behavior can be changed by configuring the ``dump.dump_destination``
 option. Read more about this and other options in
 :doc:`the DebugBundle configuration reference </reference/configuration/debug>`.
 
+.. tip::
+
+    .. versionadded:: 3.3
+        The local search box was introduced in Symfony 3.3.
+
+    If the dumped contents are complex, consider using the local search box to
+    look for specific variables or values. First, click anywhere on the dumped
+    contents and then press ``Ctrl. + F`` or ``Cmd. + F`` to make the local
```

Comment:
I think Sphinx is supposed to support keystrokes sequences: http://www.sphinx-doc.org/en/stable/markup/inline.html#role-kbd

What would it look like with this?